### PR TITLE
Translate V1alpha3 routing introduction blog to Chinese

### PR DESCRIPTION
**Pull request by zhaohuabing**
@rootsongjc @fleeto please help review.
Thanks

### CODE REVIEW

File: content_zh/blog/2018/v1alpha3-routing.md [hunk 1]
```diff
@@ -0,0 +1,318 @@
...
+
+1. 可以在 `VirtualService` 配置中表示多个匹配条件，从而减少对冗余的规则设置。
+
+1. 每个服务版本都有一个名称（称为服务子集）。 属于某个子集的一组 Pod/VM 在 `DestinationRule` 定义，具体定义参见下节。
+
+1. 通过使用带通配符前缀的 DNS 来指定 `VirtualService` 的 host，可以创建单个规则以作用于所有匹配的服务。 例如，在 Kubernetes 中，在 'VirtualService' 中使用 `*.foo.svc.cluster.local` 作为 host ,可以对 `foo` 命名空间中的所有服务应用相同的重写规则。
+
+### DestinationRule
+
+[DestinationRule](/docs/reference/config/istio.networking.v1alpha3/#DestinationRule) 配置将流量转发到服务时应用的策略集。 这些策略应由由服务提供者撰写，用于描述断路器，负载均衡设置，TLS 设置等。
```
Comment (rootsongjc) [inline content_zh/blog/2018/v1alpha3-routing.md]: 这些策略应由由服务提供者撰写 -> 这些策略应由服务提供者撰写

Comment (zhaohuabing) [reply content_zh/blog/2018/v1alpha3-routing.md]: Thanks, it's been fixed.

**Comment by geeknoid** [comment]
Hi, I did some refactoring and the file structure is a bit different now. Please rename the file to index.md and move it into a subdirectory, like the rest of the content of the site.
Please rebase and push again. Sorry for the inconvenience.

**Comment by zhaohuabing** [comment]
No problem, I'll do that.

**Comment by zhaohuabing** [comment]
Something went wrong and this pull request is messy after rebasing. I'll close this PR and create a new one.

**Pull request closed without merging**